cdc/entry/mounter.go

```diff
@@ -219,29 +219,38 @@ func (m *mounterImpl) unmarshalRowKVEntry(tableInfo *model.TableInfo, rawKey []b
 	}, nil
 }
 
-const (
-	ddlJobListKey         = "DDLJobList"
-	ddlAddIndexJobListKey = "DDLJobAddIdxList"
-)
+// IsLegacyFormatJob return true if the job is from queue.
```

Comment:
```suggestion
// IsLegacyFormatJob returns true if the job is from queue.
```
Please specify what "queue" you mean.